Configs/.config/hyprdots/scripts/volumecontrol.sh

```diff
@@ -60,11 +60,11 @@ action_playerctl ()
 while getopts iop: DeviceOpt
 do
     case "${DeviceOpt}" in
-    i) nsink=$(pamixer --list-sources | grep "_input." | tail -1 | awk -F '" "' '{print $NF}' | sed 's/"//')
+    i) nsink=$(pamixer --list-sources | grep -v "Source" | tail -1 | awk -F '" "' '{print $NF}' | sed 's/"//')
```

Comment:
this is not working as expected as its showing a source that does not have an input, not sure how to check the active device from `pamixer`. but anyways this works partially -
```
nsink=$(pamixer --list-sources | awk -F '"' 'END {print $(NF - 1)}')
```

Reply:
``` -v ``` flag inverts the output effectively avoidig the  "Source" string